### PR TITLE
In absence of GOVUK_APP_DOMAIN use dev.gov.uk

### PR DESCRIPTION
**Pull request by vinayvinay**
useful in development and test environments and keeps it consistent with Plek.rb.

### CODE REVIEW

**Comment by alext** [comment]
I'm concerned that this could result in the `dev.gov.uk` domain leaking into production.  For the ruby version we have the `RAILS_ENV` and `RACK_ENV` env variables to allow us to detect whether we're in dev or prod.  We don't really have an equivalent for Go apps.

What's the actual use case for this?

**Comment by alext** [comment]
Although thinking about it, for ruby apps, when not run under `govuk_setenv`, `RAILS_ENV` won't be set then either, so they'll have the same potential problem.

File: go/plek.go [hunk 1]
```diff
@@ -7,6 +7,8 @@ import (
 	"strings"
 )
 
+const DevDomain = "dev.gov.uk"
```
Comment (alext) [inline go/plek.go]: As this is an exported constant, it needs a godoc comment, otherwise `golint` will complain.

**Comment by vinayvinay** [comment]
@alext: need is to not have to set GOVUK_APP_DOMAIN during development or testing while using plek.go.

File: go/plek.go (hunk 1 above)
Comment (vinayvinay) [inline go/plek.go]: @alext: i've changed it to be a non-exported field, because it doesn't need to be exported.

Comment (alext) [inline go/plek.go]: Good call.

**Pull request merged**